core/src/main/java/org/apache/carbondata/core/datastore/chunk/reader/measure/v3/CompressedMeasureChunkFileBasedReaderV3.java

```diff
@@ -220,13 +223,25 @@ public CompressedMeasureChunkFileBasedReaderV3(BlockletInfo blockletInfo, String
       valueEncodeMeta.add(CarbonUtil
           .deserializeEncoderMetaNew(measureColumnChunk.getEncoder_meta().get(i).array()));
     }
-    WriterCompressModel compressionModel = CarbonUtil.getValueCompressionModel(valueEncodeMeta);
-    ValueCompressionHolder values = compressionModel.getValueCompressionHolder()[0];
+
+    MeasurePageStatistics stats = CarbonUtil.getMeasurePageStats(valueEncodeMeta);
```

Comment:
it seems this code is repeated in all readers, so better move abstract class or utility